conjur/argument_parser/_hostfactory_parser.py

```diff
@@ -54,13 +57,16 @@ def _create_hostfactory_parser(self):
 
     @staticmethod
     def _add_hostfactory_create(hostfactory_subparser: ArgparseWrapper):
+        #Needs TW review
         hostfactory_create_name = 'create - Create token for creating hosts ' \
                                   'with restrictions or creates a host using the Host Factory'
```

Comment:
```suggestion
                                  'with restrictions or create a host using the Host Factory'
```

Not sure what exaclty the meaning is here. Let's discuss